setup.cfg

```diff
@@ -9,8 +9,10 @@ description-content-type = text/x-rst; charset=UTF-8
 zip_safe = false
 install_requires =
     canonicaljson
-    connexion[swagger-ui]
+    fastapi
+    uvicorn
     ga4gh.vrs[extras]==0.7.6
+    ga4gh.vrsatile.pydantic ~= 0.0.11
```

Comment:
Should we specify in the readme that AnyVar supports VRS 1.2.X?

Reply:
👍 I have a separate branch going that overhauls the README and provides docs, we definitely should make it clear what dependencies/tool associations are assumed